src/mingw.zig

```diff
@@ -15,7 +15,66 @@ pub const CRTFile = enum {
     dllcrt2_o,
     mingw32_lib,
     mingwex_lib,
-    uuid_lib,
+
+    uuid,
+    ws2_32,
+    dinput,
+    bits,
+    shell32,
+    dmoguids,
+    dxerr8,
+    dxerr9,
+    mfuuid,
+    msxml2,
+    msxml6,
+    amstrmid,
+    wbemuuid,
+    wmcodecdspuuid,
+    windowscodecs,
+    dxguid,
+    ksuser,
+    ksguid,
+    locationapi,
+    sapi,
+    sensorsapi,
+    portabledeviceguids,
+    taskschd,
+    dloadhelper,
+    strmiids,
+    gdiplus,
+    adsiid,
```

Comment:
What's the purpose of adding all these additional CRTFile entries? Can't we just toss the source files into mingwex.lib?